README.md

```diff
@@ -60,6 +60,9 @@ constructor)
 - `stream` - if video/voice is desired, pass stream returned from
 `getUserMedia`
 
+**NOTE:** It's been observed in Chrome that you cannot establish a connection between more than 2 local peers.
+You can workaround this by opening other tabs in incognito mode. It's unclear which browsers are affected but
+this condition has been observed in Chrome 52 and 53.
```

Comment:
I would rather remove the last sentence: 'It's unclear which browsers are affected but this condition has been observed in Chrome 52 and 53.' Let's keep it terse!

Otherwise I think that's sensible!

Reply:
@pguth I'm fine with removing redundant lines, but having details on exactly what versions it has been noticed in is useful.

A 2nd attempt:

```
**NOTE:** It's been observed in Chrome (v52+) that you cannot establish a connection between more than 2 local peers.
+You can workaround this by opening other tabs in incognito mode.
```

thoughts?

Reply:
That would be nice :)